src/main/java/hellos/Hellos.java

```diff
@@ -23,7 +24,12 @@ public String generateOutput() {
     return builder.toString();
   }
 
-  private String malenaSaysHello() {
+
+  private String jackSaysHello() {
+    return "Jack says 'Hello!'\n";
+  }
+  
+    private String malenaSaysHello() {
```

Comment:
Really small thing, but it looks like you have indented the malenaSaysHello method after putting in your method

Reply:
This is important observation – thanks to @KenBroden for noting this.